tests/test_project.py

```diff
@@ -2440,7 +2440,7 @@ def test_get_job_symlink_other_project(self):
         job_a.init()
         job_b = project_b.open_job({"b": 1})
         job_b.init()
-        symlink_path = os.path.join(project_b.workspace(), job_a._id)
+        symlink_path = os.path.join(project_b.workspace(), job_a.id_)
         os.symlink(job_a.ws, symlink_path)
```

Comment:
I think this one should be reverted

Reply:
Thank you very much - this has been sorted :)